### DISCUSSION TITLE
遇到行间距太近导致无法被detect到

### DISCUSSION THREAD

**Discussion by songpeng22** [PaddlePaddle/PaddleOCR#13618] [Q&A]
项目里遇到图中文字的行间距比较近而无法识别的问题，我看到FAQ里有提到这个问题，但我没能在issues里面搜索到相关的内容，能扩展说一下吗：

FAQ里面：
Q： 文本行较紧密的情况下如何准确检测？
A：使用基于分割的方法，如DB，检测密集文本行时，最好收集一批数据进行训练，并且在训练时，并将生成二值图像的shrink_ratio参数调小一些。

我现在已知的情况，配置文件里（如ch_PP-OCRv4_det_cml.yml），MakeBorderMap和MakeShrinkMap都有shrink_ratio参数，是改这两个参数吗?
还是在python的paddleocr模块里，进入代码修改生成二值图时的shrink_ratio（代码的位置我还没找到）,然后再进行训练 ?

**Comment by GreatV** [comment]
是的

**Reply by songpeng22** [reply]
大佬能展开说一下吗，具体是改哪一处

**Reply by GreatV** [reply]
配置文件改一下
